<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>filtrowindow</class>
 <widget class="QDialog" name="filtrowindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="filtro_button">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>91</y>
     <width>321</width>
     <height>411</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#filtro_button {
background-image: url(:/diagrama/screen800x600/diagrama/Filtro 5SDF portátil_1.png);
border:none;
background-repeat: none;
background-position: center;
} </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="go_to_main">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>260</y>
     <width>31</width>
     <height>71</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#go_to_main {
background-image: url(:/iconos/images/Iconos/Flecha_izquierda.png);
border:none;
background-repeat: none;
background-position: center;
} </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_quimicos">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>560</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#pb_quimicos { 
background-image: url(:/iconos/screen800x600/iconos/Matraz azul.png);
background-color:transparent;
border: none; 
background-repeat: none;
background-position: center;
} </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="active_param_label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>580</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Phetsarath OT</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#active_param_label { 
background-color: transparent;
border: none; 
background-repeat: none;
 } </string>
   </property>
   <property name="text">
    <string>Parámetros</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_fisicos">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>560</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#pb_fisicos { 
background-image: url(:/iconos/screen800x600/iconos/Medidor azul.png);
background-color:transparent;
border: none; 
background-repeat: none;
background-position: center;
} </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="bottom_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>529</y>
     <width>766</width>
     <height>70</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#bottom_bar { 
background-color: transparent;
border-image: url(:/barras/images/Barras/Parametros_azul.png); background: none; 
border: none; 
background-repeat: none;
 } </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>530</y>
     <width>70</width>
     <height>67</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton { 
background-color: transparent;
border-image: url(:/iconos/images/Iconos/Menu_azul.png); 
background: none; 
border: none; 
background-repeat: none;
 } 
#pushButton:pressed {
border-image: url(:/iconos/images/Iconos/Menu_azul.png); 
 }
#pushButton:checked{
 border-image: none;
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_electricos">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>560</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#pb_electricos { 
background-image: url(:/iconos/screen800x600/iconos/Clavija azul.png);
background-color:transparent;
border: none; 
background-repeat: none;
background-position: center;
} </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="asa_logo">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>550</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#asa_logo{ 
background-image: url(:/iconos/screen800x600/iconos/Logo.png);
border: none; 
background-repeat: none;
background-position: center;
} </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_dia">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>28</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Dia</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="prof_pic">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#prof_pic { 
background-image: url(:/iconos/screen800x600/iconos/Prof pic azul.png);
background-color:transparent;
border: none; 
background-repeat: none;
background-position: center;
} </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_hora">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hora</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="prof_label">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>20</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Sin usuario</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>100</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>130</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>230</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="modulo_1">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>181</width>
     <height>221</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#modulo_1 { 
border: none; 
background-color: transparent;
background-repeat: none;
background-position: center;
} </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="top_menu_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#top_menu_2 { 
background-color: transparent;
background-image: url(:/iconos/screen800x600/iconos/Bitacora azul.png);
background-position: center;
border: none; 
background-repeat: none;
 } </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="top_menu_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>1</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#top_menu_3 { 
background-color: transparent;
background-image: url(:/iconos/screen800x600/iconos/Monitor1.png);
background-position: center;
border: none; 
background-repeat: none;
 } </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="top_menu_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>34</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#top_menu_6 { 
background-color: transparent;
background-image: url(:/iconos/screen800x600/iconos/Encendido azul.png);
background-position: center;
border: none; 
background-repeat: none;
 } </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="top_menu_1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#top_menu_1 { 
background-color: transparent;
background-image: url(:/iconos/screen800x600/iconos/Campana.png);
border: none; 
background-repeat: none;
background-position: center;

 } </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="label_title">
   <property name="geometry">
    <rect>
     <x>259</x>
     <y>-1</y>
     <width>261</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_title
{
background-color: transparent;
color: black;
border: none;
}</string>
   </property>
   <property name="text">
    <string>Diagrama</string>
   </property>
  </widget>
  <widget class="QLabel" name="top_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>2</y>
     <width>759</width>
     <height>27</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#top_bar { 
background-color: transparent;
background-image: url(:/barras/screen800x600/barras/Indice diagrama limpio.png);
border: none; 
background-repeat: none;
 } </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="modulo_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>211</width>
     <height>131</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#modulo_2 { 
border: none; 
background-color: transparent;
background-repeat: none;
background-position: center;
background-image: url(:/diagrama/screen800x600/diagrama/motor_filtro.png);
} </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget_6" native="true">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <zorder>top_bar</zorder>
  <zorder>filtro_button</zorder>
  <zorder>go_to_main</zorder>
  <zorder>bottom_bar</zorder>
  <zorder>label_dia</zorder>
  <zorder>label_hora</zorder>
  <zorder>prof_label</zorder>
  <zorder>prof_pic</zorder>
  <zorder>active_param_label</zorder>
  <zorder>pb_quimicos</zorder>
  <zorder>pb_electricos</zorder>
  <zorder>pb_fisicos</zorder>
  <zorder>asa_logo</zorder>
  <zorder>widget_3</zorder>
  <zorder>widget_2</zorder>
  <zorder>widget</zorder>
  <zorder>modulo_1</zorder>
  <zorder>pushButton</zorder>
  <zorder>top_menu_2</zorder>
  <zorder>top_menu_3</zorder>
  <zorder>top_menu_6</zorder>
  <zorder>top_menu_1</zorder>
  <zorder>label_title</zorder>
  <zorder>modulo_2</zorder>
  <zorder>widget_6</zorder>
  <zorder>widget_4</zorder>
  <zorder>widget_5</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
